packages/clerk-js/src/utils/clerkRequirements.ts

```diff
@@ -0,0 +1,22 @@
+import { Clerk, ClerkOptions, EnvironmentResource } from '@clerk/types';
+
+export type ClerkRequirement = (
```

Comment:
Note sure if that'd make more sense:

```suggestion
export type ComponentGuardCondition = (
```

Reply:
Done